### ISSUE TITLE
Role combobox added on Input element is not announced properly in IE11 and JAWs

### ISSUE THREAD

**Issue by pavandixit93** [FreedomScientific/VFO-standards-support#475]
### Summary
We have one usecase in which we need to add `role="combobox` on `Input` element. While testing `JAWs+IE11` combination we found that JAWs does not announce it correctly.
CASE1: When user focus on Input box having `role="combobox"` and `aria-label="Username"` It only announces `Username edit type and text",` But when tested on `JAWS+Chrome` It announces `Username combobox, Press UP or DOWN keys to change the selection...`. 
CASE 2: I also observed that if i add `readonly="true"` on Input element then `JAWs+IE11` announces that whole message i.e. `Press UP or DOWN keys to change the selection`, 

> Brief description of the issue. Please list any specific steps.
> 
> Example:
> 1. Open Internet Explorer(IE11) and Go to https://stackblitz.com/edit/jaws-ie11-role-combobox-issue?file=index.html
> 2. Focus on `Username` Input box, listen what JAWs announces
> 3. Focus on `Email`  Input box, listen what JAWs announces
> 4. Now Follow the similar steps but using Chrome + JAWs

### Expected result
JAWs should announce the context correctly when role=combobox is used on Input element in IE11.
> What did you expect to happen?

### Actual result
JAWs does not announcing the context correctly when role=combobox is used on Input element in IE11.
> What actually did happen?

### Example

> A test case that demonstrates the issue - provide linked test case, CodePen, JSbin, etc...

**Comment by JAWS-test** [comment]
See: https://github.com/FreedomScientific/VFO-standards-support/issues/399

combobox has never worked in IE 11 with JAWS and will - according to my forecast - never work. IE 11 is outdated and buggy, and Freedom Scientific also seems to think that they no longer need to support IE 11. That old versions of JAWS did not support combobox is, of course, a JAWS bug. In my opinion there are only 2 solutions for the problem:

* do not use IE 11
* do not use combobox with IE 11

**Comment by pavandixit93** [comment]
@JAWS-test Thank you and i totally agree with you but customer uses IE11. I prefer Chrome/Firefox/Edge. So will be tough to convince them 😞

**Comment by stevefaulkner** [comment]
@pavandixit93, @JAWS-test is correct and this issue will probably not be fixed. Have a look at https://alphagov.github.io/accessible-autocomplete/examples/ for some examples that do work in IE despite the lack of support.

**Comment by JAWS-test** [comment]
In IE 11 the [examples](https://alphagov.github.io/accessible-autocomplete/examples) work better than combobox, but I would not call them accessible.

* Input field with autocomplete function is not the same as a comboboxes (e.g. with autocomplete I have to enter characters before entries are displayed, with combobox I don't)
* The output of the list entries is very verbose: 

"Bangladesh 
4 of 89
4 of 89
[Operating note regarding arrow keys]
89 results are available. Bangladesh 4 of 89 is highlighted"

**Comment by stevefaulkner** [comment]
Closing this issue as wontfix as its IE only
